ankerctl.py

```diff
@@ -81,6 +82,9 @@ def main(ctx, verbose, quiet, insecure):
 
     env.upgrade_config_if_needed()
 
+    env.printer = printer
+    log.debug(f"Using printer [{env.printer}]");
```

Comment:
Naming-wise, using the word `printer` to refer to an *index* is a little confusing, since everywhere else in the code, we use `printer` to refer to a printer-related structure (such as a specific printer config).

I'd be ok with just `index`, if you want to keep it short, but otherwise something like `printer_index`? Suggestions welcome.

Reply:
Updated this to `printer_index`